Sming/SmingCore/Network/WebSocket.cpp

```diff
@@ -75,3 +75,13 @@ void WebSocket::close()
 {
 	connection->close();
 }
+
+void WebSocket::setUserData(void* _userData)
```

Comment:
Keep the coding style consistent. Rename '_userData' to 'userData'. Then use `this->userData = userData`